app/modules/organizations/models.py

```diff
@@ -79,20 +80,33 @@ def _process_members(self, members):
             #    self.members.append(user)
```

Comment:
This bit also needs to be restored now you have the relationship working, this is the part that stitches them together on the sync

Reply:
This is a bit more complicated now.   We store the relationship as a `HoustonModel` with its own separate database table so that things like created and updated timestamps are preserved.  There is a new test in `tests/modules/organizations/test_models.py` that shows how to add memberships now.  I'd also request you take a look at how the properties for `User.memberships` and `Organization.members` is defined.